platform-tests/bin/java

```diff
@@ -23,6 +23,7 @@
 set -exo pipefail
 #JAVA_CALL="java -DsourceCodeIndexerPath=/home/agibsonccc/Documents/GitHub/deeplearning4j -javaagent:/home/agibsonccc/.m2/repository/org/nd4j/nd4j-log-analyzer/1.0-SNAPSHOT/nd4j-log-analyzer-1.0-SNAPSHOT.jar "
 JAVA_CALL="java "
+source /home/agibsonccc/Documents/GitHub/deeplearning4j/platform-tests/bin/env.sh
```

Comment:
It seems wrong to keep your path here.

Reply:
Ah yeah this was left over from some troubleshooting. I wasn't sure if the variables were working or not so I was double checking paths..